src/main/scala/scorex/api/http/ApiError.scala

```diff
@@ -23,7 +23,7 @@ trait ApiError {
 
 object ApiError {
   def fromValidationError(e: ValidationError): ApiError = e match {
-    case ValidationError.InvalidAddress => InvalidAddress
+    case ValidationError.InvalidAddress(_) => InvalidAddress
```

Comment:
We should probably expose address validation error to API consumer

Reply:
Decided not to break API